cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/Filters/FilterCollection/index.tsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import Button from '@material-ui/core/Button';
+import withStyles, { StyleRules, WithStyles } from '@material-ui/core/styles/withStyles';
+import { IPropertyShowConfig } from 'components/ConfigurationGroup/types';
+import WidgetWrapper from 'components/ConfigurationGroup/WidgetWrapper';
+import If from 'components/If';
+import FilterConditionInput from 'components/PluginJSONCreator/Create/Content/Filters/FilterCollection/FilterConditionInput';
+import FilterShowlistInput from 'components/PluginJSONCreator/Create/Content/Filters/FilterCollection/FilterShowlistInput';
+import { ICreateContext } from 'components/PluginJSONCreator/CreateContextConnect';
+import * as React from 'react';
+import uuidV4 from 'uuid/v4';
+
+const styles = (theme): StyleRules => {
+  return {
+    filterInput: {
+      display: 'block',
+      marginLeft: 'auto',
+      marginRight: 'auto',
+      marginTop: '30px',
+      marginBottom: '30px',
```

Comment:
consolidate these to the `margin` property

Reply:
Resolved